staking/availability/measure.go

```diff
@@ -185,6 +185,11 @@ func ComputeCurrentSigning(
 	}
 
 	s1, s2 := numeric.NewDecFromBigInt(signed), numeric.NewDecFromBigInt(toSign)
+	if s2.IsNil() || s2.IsZero() {
```

Comment:
Should remove this, if this is violated, then we should find out asap and I did from the stack trace last time (that something was iffy)

Reply:
ok sure.